openwisp_monitoring/device/models.py

```diff
@@ -68,9 +68,9 @@ def data_user_friendly(self):
                 remove.append(interface)
                 continue
             # human readable mode
-            interface['wireless']['mode'] = interface['wireless']['mode'].replace(
-                '_', ' '
-            )
+            # interface['wireless']['mode'] = interface['wireless']['mode'].replace(
+            #     '_', ' '
+            # )
```

Comment:
Why is it commented? If it's not useful then let's remove it

Reply:
I was testing another template which didn't have `wireless` in `Devicedata` but it is useful